Jenkinsfile

```
@@ -616,6 +622,54 @@ def withBeatsEnvWin(Closure body){
   }
 }
 
+def dumpFilteredEnivronment(){
```

I am not the only one who writes `environment` always in the wrong way 😃 

```suggestion
def dumpFilteredEnvironment(){
```

---

Spelling will be the death of me! Thanks, @kuisathaverat